### PR TITLE
feat: added sink batch size config for sink concurrency

### PR DESCRIPTION
**Pull request by sumslim**

### CODE REVIEW

File: agent/agent.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-const defaultBatchSize = 1
+const defaultBatchSize = 10
```
Comment (batrov) [inline agent/agent.go]: can make this configurable

File: config/meteor.yaml.sample [hunk 2]
```diff
@@ -5,4 +5,5 @@ STOP_ON_SINK_ERROR: false
 APP_NAME: meteor
 OTEL_ENABLED: false
 OTEL_COLLECTOR_ADDR: "localhost:4317"
-OTEL_TRACE_SAMPLE_PROBABILITY: 1
\ No newline at end of file
+OTEL_TRACE_SAMPLE_PROBABILITY: 1
+BATCH_SIZE: 10
```
Comment (haveiss) [inline config/meteor.yaml.sample]:
```suggestion
SINK_BATCH_SIZE: 10
```

File: agent/agent_test.go [hunk 3]
```diff
@@ -728,6 +728,11 @@ func TestAgentRun(t *testing.T) {
 		data := []models.Record{
 			models.NewRecord(&v1beta2.Asset{}),
 		}
+		batchedData := []models.Record{
+			models.NewRecord(&v1beta2.Asset{}),
+			models.NewRecord(&v1beta2.Asset{}),
+			models.NewRecord(&v1beta2.Asset{}),
+		}
```
Comment (haveiss) [inline agent/agent_test.go]: why this sink 3 data?

Comment (sumslim) [reply agent/agent_test.go]: fixed

**Review by haveiss — APPROVED** [review]

**Review by utsav14nov — APPROVED** [review]

**Pull request merged**